### PR TITLE
query linked ethereum addresses

### PR DESCRIPTION
**Pull request by suinuj**
The pr add the CLI command to get the ethereum addresses linked to an accountId.

### CODE REVIEW

File: app-libs/stf/src/cli.rs [hunk 1]
```diff
@@ -270,6 +271,42 @@ pub fn cmd<'a>(
 					Ok(())
 				}),
 		)
+		.add_cmd(
+			Command::new("linked-eth-addresses")
+				.description("query linked ethereum addresses for account")
```
Comment (haerdib) [inline app-libs/stf/src/cli.rs:276]:
```suggestion
				.description("query linked ethereum addresses for given account")
```

File: app-libs/stf/src/helpers.rs [hunk 2]
```diff
@@ -81,6 +81,11 @@ pub fn get_account_info(who: &AccountId) -> Option<AccountInfo> {
 	get_storage_map("System", "Account", who, &StorageHasher::Blake2_128Concat)
 }
 
+// litentry get linked etherem addresses
+pub fn get_linked_ethereum_addresses(who: &AccountId) -> Option<Vec<EthAddress>> {
```
Comment (haerdib) [inline app-libs/stf/src/helpers.rs:85]: That's a good call. If all Prefixes are placed here, it will be easy to adapt in case it's necessary :)

**Review by haerdib — COMMENTED** [review]
I think it should work like that. I can't test it because it's not compiling, but these are minor bugs I thinks.

**Comment by suinuj** [comment]
will fix the compilation error.

**Review by haerdib — APPROVED** [review]
Works great! Wasn't able to test it with an actual return value, but I issued the following call:

` ./integritee-cli -p 9994 -P 2094 trusted linked-eth-addresses //Alice --mrenclave=DiYsUzEh3nL5N8mrqGu1fVxDYi7ZPTRUdYjmE2s5ioLN` and it returned
`[]` which is correct, as we don't have an account registered yet.

On the worker side I got the following error logs:
```
[2021-11-18T08:53:21Z ERROR ita_stf::stf_sgx] Trying to get Linked ethereum addresses
[2021-11-18T08:53:21Z ERROR ita_stf::helpers] key not found in state [52, 3, cb, a2, 63, f7, e9, 33, 7d, 89, a3, 21, 1e, f7, fa, 56, 6e, d0, ce, 7a, b8, 7c, 71, ae, 5a, cb, 4a, dd, 5d, 50, 66, 58, de, 1e, 86, a9, a8, c7, 39, 86, 4c, f3, cc, 5e, c2, be, a5, 9f, d4, 35, 93, c7, 15, fd, d3, 1c, 61, 14, 1a, bd, 4, a9, 9f, d6, 82, 2c, 85, 58, 85, 4c, cd, e3, 9a, 56, 84, e7, a5, 6d, a2, 7d]
[2021-11-18T08:53:21Z ERROR ita_stf::stf_sgx] No addresses listed
```
Where the first and last one of the error logs I added for visibility local only, not in the GitHub code. But the second one can be found here: https://github.com/litentry/worker/blob/cb829a27d1486db4673f4c7acdea8c34ddb1b376/app-libs/stf/src/helpers.rs#L71.

So the call reaches the litentry pallet perfectly and returns to the client. Good work.

**Pull request merged**